master/getting-started/kubernetes/installation/integration.md

```diff
@@ -104,6 +104,13 @@ WantedBy=multi-user.target
 ```
 > Replace `<ETCD_IP>:<ETCD_PORT>` with your etcd configuration.
 
+> **NOTE:**
+>
+> When scaling to hundreds of nodes it was found that the dataplane programming
```

Comment:
I think you should drop the part about hundreds of nodes; it doesn't matter how small your cluster is.  If it's under CPU contention then we need a reservation.